src/img/icons/search.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 18 18">
+    <g>
```

Comment:
Why do you think this one has a group tag? There's only one path 😅

Reply:
I will change this , oops